src/generate-manifest.js

```diff
@@ -30,6 +30,14 @@ const generateManifest = async function (codiusVarsPath, codiusPath) {
   debug('validating generated manifest...')
   validateFinalManifest(generatedManifest)
 
+  // Optionally add an empty private vars field
```

Comment:
Can we expand upon this comment so its explained why we need this option, maybe even cite the codiusd issue #.

Reply:
Great idea. I added a more descriptive comment in one my latest commits.